asmcomp/cmmgen.ml

```diff
@@ -868,13 +873,9 @@ let curry_function n =
 
 (* Comparisons *)
 
-let transl_comparison = function
-    Lambda.Ceq -> Ceq
-  | Lambda.Cneq -> Cne
-  | Lambda.Cge -> Cge
-  | Lambda.Cgt -> Cgt
-  | Lambda.Cle -> Cle
-  | Lambda.Clt -> Clt
+let transl_int_comparison cmp = cmp
+
+let transl_float_comparison cmp = cmp
```

Comment:
These two functions are dummies now. Why not remove them, refactoring their uses?